aggregator_core/src/task.rs

```diff
@@ -479,6 +479,168 @@ impl Task {
+            self.helper_aggregator_endpoint.clone(),
+            self.query_type,
+            self.vdaf.clone(),
+            self.vdaf_verify_key.clone(),
+            self.max_batch_query_count,
+            self.task_expiration,
+            self.report_expiry_age,
+            self.min_batch_size,
+            self.time_precision,
+            self.tolerable_clock_skew,
+            self.hpke_keys.values().cloned().collect::<Vec<_>>(),
+            AggregatorTaskParameters::Leader {
+                aggregator_auth_token: self
+                    .aggregator_auth_token
+                    .clone()
+                    .ok_or(Error::InvalidParameter("no aggregator auth token in task"))?,
+                collector_auth_token: self
+                    .collector_auth_token
+                    .clone()
+                    .ok_or(Error::InvalidParameter("no collector auth token in task"))?,
+                collector_hpke_config: self
+                    .collector_hpke_config
+                    .clone()
+                    .ok_or(Error::InvalidParameter("no collector HPKE config in task"))?,
+            },
+        )
+    }
+
+    /// Render the helper aggregator's view of this task.
+    pub fn helper_view(&self) -> Result<AggregatorTask, Error> {
+        AggregatorTask::new(
+            self.task_id,
+            self.leader_aggregator_endpoint.clone(),
+            self.query_type,
+            self.vdaf.clone(),
+            self.vdaf_verify_key.clone(),
+            self.max_batch_query_count,
+            self.task_expiration,
+            self.report_expiry_age,
+            self.min_batch_size,
+            self.time_precision,
+            self.tolerable_clock_skew,
+            self.hpke_keys.values().cloned().collect::<Vec<_>>(),
+            AggregatorTaskParameters::Helper {
+                aggregator_auth_token: self
+                    .aggregator_auth_token
+                    .clone()
+                    .ok_or(Error::InvalidParameter("no aggregator auth token in task"))?,
+                collector_hpke_config: self
+                    .collector_hpke_config
+                    .clone()
+                    .ok_or(Error::InvalidParameter("no collector HPKE config in task"))?,
+            },
+        )
+    }
+
+    /// Render a taskprov helper aggregator's view of this task.
+    pub fn taskprov_helper_view(&self) -> Result<AggregatorTask, Error> {
+        AggregatorTask::new(
+            self.task_id,
+            self.leader_aggregator_endpoint.clone(),
+            self.query_type,
+            self.vdaf.clone(),
+            self.vdaf_verify_key.clone(),
+            self.max_batch_query_count,
+            self.task_expiration,
+            self.report_expiry_age,
+            self.min_batch_size,
+            self.time_precision,
+            self.tolerable_clock_skew,
+            self.hpke_keys.values().cloned().collect::<Vec<_>>(),
+            AggregatorTaskParameters::TaskProvHelper,
```

Comment:
[Elsewhere we've used `Taskprov`](https://github.com/divviup/janus/blob/main/aggregator_api/src/models.rs#L210) instead of `TaskProv` (i.e. lowercase P).

Whichever one works for me, but we should be consistent. It also wasn't introduced in this PR so doesn't have to be resolved here. Can proceed, then maybe a PR after the rest that does `find | sed`.

Reply:
That makes sense. I'm going to change the name in a later PR so that I don't have to rebase the existing stack.